caiman/source_extraction/volpy/volpy.py

```diff
@@ -103,42 +128,42 @@ def fit(self, n_processes=None, dview=None):
                 results_part = dview.map_async(volspike, args_in).get(4294967)
             elif dview is not None:
                 results_part = dview.map_sync(volspike, args_in)
             else:
                 results_part = list(map(volspike, args_in))
-                    
             results = results + results_part
         
-        N = len(results)
-        print(N)
-        self.estimates['spikeTimes'] = [results[i]['spikeTimes'] for i in range(N)]
-        self.estimates['trace'] = [results[i]['yFilt'] for i in range(N)]
-        self.estimates['recons_signal'] = [results[i]['recons_signal'] for i in range(N)]
-        self.estimates['spatialFilter'] = [results[i]['spatialFilter'] for i in range(N)]
-        self.estimates['templates'] = [results[i]['templates'] for i in range(N)]
-        self.estimates['cellN'] = [results[i]['cellN'] for i in range(N)]
-        self.estimates['templates'] = [results[i]['templates'] for i in range(N)]
-        self.estimates['snr'] = [results[i]['snr'] for i in range(N)]
-        self.estimates['num_spikes'] = [results[i]['num_spikes'] for i in range(N)]
-        self.estimates['passedLocalityTest'] = [results[i]['passedLocalityTest'] for i in range(N)]
-        self.estimates['low_spk'] = [results[i]['low_spk'] for i in range(N)]
-        self.estimates['weights'] = [results[i]['weights'] for i in range(N)]
-        self.estimates['bwexp'] = [results[i]['bwexp'] for i in range(N)]
-        
+        self.estimates['cell_n'] = np.array([results[i]['cell_n'] for i in range(N)])
+        self.estimates['t'] = np.array([results[i]['t'] for i in range(N)])
+        self.estimates['ts'] = np.array([results[i]['ts'] for i in range(N)])
+        self.estimates['t_rec'] = np.array([results[i]['t_rec'] for i in range(N)])
+        self.estimates['t_sub'] = np.array([results[i]['t_sub'] for i in range(N)])
+        self.estimates['spikes'] = np.array([results[i]['spikes'] for i in range(N)])
+        self.estimates['low_spikes'] = np.array([results[i]['low_spikes'] for i in range(N)])
+        self.estimates['num_spikes'] = np.array([results[i]['num_spikes'] for i in range(N)])
+        self.estimates['templates'] = np.array([results[i]['templates'] for i in range(N)])
+        self.estimates['snr'] = np.array([results[i]['snr'] for i in range(N)])
+        self.estimates['thresh'] = np.array([results[i]['thresh'] for i in range(N)])
+        self.estimates['spatial_filter'] = np.array([results[i]['spatial_filter'] for i in range(N)])
+        self.estimates['weights'] = np.array([results[i]['weights'] for i in range(N)])
+        self.estimates['locality'] = np.array([results[i]['locality'] for i in range(N)])
+        self.estimates['context_coord'] = np.array([results[i]['context_coord'] for i in range(N)])
+        self.estimates['F0'] = np.array([results[i]['F0'] for i in range(N)])
+        self.estimates['dFF'] = np.array([results[i]['dFF'] for i in range(N)])
```

Comment:
You could probably iterate over all of these more elegantly given that the keynames are the same